chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+# Automatically recognising speech
+
+In this section, we’ll take a look at how Transformers can be used to convert spoken speech into text, a task known _speech recognition_.
+
+TODO: insert Speech -> Model -> Text diagram
+
+Speech recognition, also known as automatic speech recognition (ASR) or speech-to-text (STT), is one of the most popular and exciting spoken language processing tasks. It’s used in a wide range of applications, including dictation, voice assistants, video captioning and meeting transcriptions.
+
+You’ve probably made use of a speech recognition system many times before without realising! Consider the digital assistant in your smartphone device (Siri, Google Assistant, Alexa). When you use these assistants, the first thing that they do is transcribe your spoken speech to written text, ready to be used for any downstream tasks.
+
+Image: https://images.app.goo.gl/J6SU6RYryXXVT9uj7
+Demo: https://huggingface.co/facebook/wav2vec2-base-960h
```

Comment:
For the demo, I suggest embedding a Gradio Space directly in the page, similar to how we did it in the NLP tasks: https://moon-ci-docs.huggingface.co/course/pr_294/en/chapter7/2?fw=pt

Reply:
(Attempted) resolve in https://github.com/huggingface/course/pull/294/commits/9d111104b597c83c0eb05f25f36544eed01eb3a0